### DISCUSSION TITLE
Walled Garden Escape (In-app direct update)

### DISCUSSION THREAD

**Discussion by du82** [status-im/status-legacy#13288] [Ideas]
# Feature Issue

### User Story
As a user I want to be able to download Status directly, instead of going through the Play store. 

### Description
I think this should be done like the way Telegram does it, where it prompts the user to install an update, while bypassing the Google Play store. We could pull the download file from F-Droid which removes the need for users to download the app from Status or an untrusted source.

My mobile apps do this for Status promotions, instead of going through the play store, Status gets downloaded directly through a typically dormant feature that allows for downloading of other apps, thus escaping the Google Play store's walled garden.

### Acceptance Criteria
* Must download Status from a trusted source (F-Droid, GitHub) and verify package hash against the official ones to detect tampering.
* Shouldn't rely on Google dependencies, and should be able to escape the walled garden that is the Google Play store.
* A "here's what's new" section would be nice to show users what changes will come before they update, and what they can do now after they update.

![Screenshot_20220314-104258_Telegram](https://user-images.githubusercontent.com/65834069/158294052-df3e3911-3c8d-489d-bf5c-4898064b27f2.png)

**Comment by horihori12** [comment]
The site is easy to read. I enjoy coming here. I also want to tell you that the soil knife is a very helpful tool for gardening. It is sharp, strong, and great for digging, planting, and soil knife cutting roots. I like how easy it is to use and how [soil knife](https://horihori.com/products/rosewood-handle-stainless-steel-hori-hori-gardening-knife) makes garden work simple. A soil knife is a must-have for every gardener’s toolbox.
